README.adoc

```diff
@@ -8,145 +8,93 @@ image:https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic%20rel
-
-
-.7+| Clean
-
-| *[default]* all
-v|`npm run clean` or
-`npm run clean \-- --all`
-| Fully resets the project
-
-| cache
-v|`npm run clean \-- --cache`
-| Clears `*npm*` cache
-
-| config
-v|`npm run clean \-- --config`
-| Resets app config & env_vars
-
-| dist
-v|`npm run clean \-- --dist`
-| Removes the build artifacts
-
-| images
-v|`npm run clean \-- --images`
-| Stops containers & removes the images
+* Use -r - to reinstall everything
+* Use -s - to start Planner in standalone mode using the runtime
+* Use -f - to set the path to Fabric8 UI directory (This requires path argument)
+* Use -p - to set the path to the Planner directory (This requires the path argument)
 
-| modules
-v|`npm run clean \-- --modules`
-| Removes all `*node*` modules
++
+For example:
+* Use scripts/run-planner.sh -r -s - to reinstall planner and run it in standalone mode.
+* Use scripts/run-planner.sh -r -f /home/User/fabric8-ui - if you wish to set the platform path. By default, the Fabric8 UI path is assumed to be Planner_Path/..
 
-| temp
-v|`npm run clean \-- --temp`
-| Removes temp files and artifacts
+. Access the service on:
+http://localhost:3000
 
+=== Standalone mode
+The Standalone Mode uses prod-preview backend by default.
 
-.3+| Tests
+. To build and run a standalone, minimal runtime implementation of the Planner using the prod-preview database use:
++
+----
+$ scripts/run-planner.sh --standalone
+----
++
+Or
++
+----
+$ scripts/run-planner.sh -s
+----
++
+. Access the service on http://localhost:8080
+. Get an auth token:
+.. Navigate to http://api.prod-preview.openshift.io/api/login/authorize?redirect=https://api.openshift.io/api/status
+.. Log in with your OSIO user name (for example, <username>+preview@redhat.com). The part after "?token_json" in the resulting url is the token. Copy this token.
+. Use the token to log in as: `localhost:8080/?<the token>`
 
-| unit
-v|`npm run tests \-- --unit`
-| Runs the unit tests
+. Navigate to prod-preview and get a space ID you want to use from the url.
+. Use that spaceId to get to a space on your local instance: http://localhost:8080/plan/list?q=(space:<spaceId>%20$AND%20typegroup.name:Scenarios)&showTree=true
 
-| [line-through]#func#
-v|`npm run tests \-- --func`
-| Runs the functional tests
+Optionally, you can change the default prod-preview backend and use external web apps by setting the environment variables for the services, for example:
 
-| [line-through]#smok#
-v|`npm run tests \-- --smok`
-| Runs the smoke tests
+* To use Fabric8 wit as the backend use the environment variable: FABRIC8_WIT_API_URL
+* To use Fabric8 Auth as the backend use the environment variable: FABRIC8_AUTH_API_URL
 
-4+| Commands with [line-through]#strikethroughs# aren't yet implemented.
+== Documentation
+The following documentation is available in the *docs* directory:
 
-|===
+*  link:docs/alternate_ways_to_run_planner.adoc[Alternate ways to run Planner]: Provides information about alternate methods to run Planner.
```

Comment:
Can we arrange the documentation as
1. `Technology Stack`
2. `Testing`
3. `Alternate ways to run Planner`
4. `Useful scripts and tasts`